doc/userdoc/contribute/code_review_guidelines.rst

```diff
@@ -13,24 +13,37 @@ meant to prevent progress, but to keep up the code quality as the
 number of developers is growing. All of this is not set in stone and
 can be discussed on the :ref:`NEST mailing lists <community>`.
 
-* In general, the rule is that each pull request needs an OK from the CI
-  platform and at least two reviewers to be merged.
-* For changes labeled “not code” or “minor” (e.g., changes in documentation,
-  fixes for typos, etc.), the release manager can waive the need for code
-  review and just accept the OK from the CI system in order to merge the request.
-* Each pull request needs to be documented by an issue in the `issue
-  tracker <https://github.com/nest/nest-simulator/issues>`_ explaining the reason
-  for the changes and the solution. The issue is also the place for discussions
-  about the code.
-* New features like SLI or PyNEST functions, neuron or synapse models need to
-  be accompanied by one or more tests written either in SLI or Python. New
-  features for the NEST kernel need a test written in SLI.
-* Each change to the code has to be reflected also in the corresponding
-  examples and documentation.
-* All source code has to be adhering to the Coding Guidelines for
-  :ref:`C++ <code_style_cpp>` in order to
-  pass the :ref:`continuous integration system <cont_integration>`.
-* All commits should be coherent and contain only changes that belong together.
-* Please check the **typeset documentation** as part of the review process. To
-  learn how to test the documentation locally offline, please check out our
-  :ref:`User documentation workflow <userdoc_workflow>`.
+* In general, the rule is that each pull request needs an OK from the
+  CI platform and at least two reviewers to be merged.
+* The two reviews have to cover the technical side (i.e., if the code
+  does the right thing and is architecturally sane) and the content
+  side (i.e., if the code is scientifically correct and fixes an
+  actual issue)
+* For changes labeled “not code” or “minor” (e.g., changes in
+  documentation, fixes for typos, etc.), the need for a second code
+  review can be waived and a single rewiew plus the OK from the CI
+  system is sufficient in order to merge the request.
+* New features like SLI or PyNEST functions, neuron or synapse models
+  need to be accompanied by one or more tests written either in SLI or
+  Python.
+* Each change to the code has to be reflected also in the
+  corresponding examples and documentation.
+* A pull request should be coherent and contain only changes that
+  belong together.
+* Please check the **typeset documentation** as part of the review
```

Comment:
```suggestion
* Please check the **typeset documentation** as part of the review
```

Do you mean the cpp guidelines, and reStructuredText etc?

Reply:
I guess. This sentence has already been there before...

Reply:
Maybe it would be better to link to the contribute/index page? and to check relevant guidelines found there?

Reply:
I somehow fail to see that connection at the moment. Could you please elaborate? For me, this point is about the fact that reviewers have to check the rendered version of documentation (if that applies) and the existing link to the documentation workflow seems appropriate to me.

Reply:
Ok I see, I misunderstood what was meant by typeset documentation. 
How about:
```suggestion
* Please check that the typeset in the output documentation looks correct as part of the review
```

Reply:
Maybe like this?
```suggestion
* Please also check that the typesetting of the documentation looks correct.
```

Reply:
:+1: